### PR TITLE
Add sample test packages

### PR DESCRIPTION
**Pull request by mtojek**
Closes https://github.com/elastic/elastic-package/issues/108

_I'm opening as draft as there is something wrong with system tests runner. Let's wait for the CI._

### CODE REVIEW

**Comment by mtojek** [comment]
Stack trace:

```
➜  apache git:(add-sample-test-package) pwd
/Users/marcin.tojek/go/src/github.com/elastic/elastic-package/test/packages/apache
➜  apache git:(add-sample-test-package) elastic-package test system
Run system tests for the package
2020/09/18 12:24:54  INFO setting up service...
Error: error running package system tests: could not create service runner: unable to find service runner
```

**Comment by ycombinator** [comment]
> Error: error running package system tests: could not create service runner: unable to find service runner

Maybe because there's nothing under `_dev/deploy`?

**Comment by mtojek** [comment]
Yes, but there are multiple integrations that don't have any `_dev`.

File: scripts/test-check-packages.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+for d in test/packages/*/; do
+  (
+    cd $d
+    elastic-package check -v
+    elastic-package test -v
+  )
+done
+
+elastic-package stack down -v
```
Comment (ycombinator) [inline scripts/test-check-packages.sh:17]: I wish bash had `defer`s 🙂

**Review by ycombinator — APPROVED** [review]
LGTM. Great idea to add these!

File: scripts/test-check-packages.sh (hunk 1 above)
Comment (mtojek) [reply scripts/test-check-packages.sh:17]: Hm.. I will try with "trap".

Comment (mtojek) [reply scripts/test-check-packages.sh]: I think it works as intended :)

**Pull request merged**